src/sdk/model/org/__tests__/org.test.int.ts

```diff
@@ -96,3 +96,51 @@ test('Can refresh Org', async() => {
     });
   });
 });
+
+test('Can get org edges', async() => {
+  const org = await Org.getOrg(inventoryOrg.uuid);
```

Comment:
It can be a good idea to add a `expect.assertions(num)` when num is the actual number of test expected. Specially when we have promise inside promise.

Reply:
Nevermind. This is not needed there.